### DISCUSSION TITLE
Keybinds to change colours:

### DISCUSSION THREAD

**Discussion by droshux** [xournalpp/xournalpp#5649] [Q&A]
UPDATE: The number keys are the ones that change colour! This isn't written in the toolbar colours page anywhere, I'll edit this myself sometime but it should be there. 

I write on a touchscreen laptop with the toolbar hidden and sometimes I keep accidentally changing my pen colour and I assume this is because I'm hitting some keybind to change the pen colour. This isn't me hitting `CTRL+SHIFT+D` because the colour is not my default for the pen tool (it's the dark green).

Is this what's happening? If it is I can't find any info on it on the wiki. If it isn't does anyone have a clue why this happens?

It would be cool if I could change colour using a keybind so I hope that actually is what is happening.

**Comment by bhennion** [comment]
Depending on the version you are using, you can switch between colors by hitting one of the number keys.
